tests/test_main.py

```diff
@@ -196,52 +203,54 @@ def test_version(capsys):
 
 @pytest.mark.parametrize("explicit_file", [True, False])
 def test_error_message_environment_file_not_found(
-    capsys, tmpdir, explicit_file, monkeypatch
-):
-    monkeypatch.chdir(str(tmpdir))
+    capsys, tmp_path: Path, explicit_file, monkeypatch
+) -> None:
+    monkeypatch.chdir(tmp_path)
     args = ["--file", "invalid.devenv.yml"] if explicit_file else []
     expected_name = "invalid.devenv.yml" if explicit_file else "environment.devenv.yml"
     assert devenv.main(args) == 1
     out, err = capsys.readouterr()
     assert out == ""
-    assert err == 'File "{}" does not exist.\n'.format(str(tmpdir / expected_name))
+    assert err == 'File "{}" does not exist.\n'.format(str(tmp_path / expected_name))
 
 
-def test_get_env_directory(mocker, tmpdir):
-    import os
-
-    env_0 = tmpdir.join("0/envs/my_env").ensure(dir=1)
-    conda_meta_env_0 = tmpdir.join("0/envs/my_env/conta-meta").ensure(dir=1)
+def test_get_env_directory(mocker, tmp_path: Path) -> None:
+    env_0 = tmp_path / "0/envs/my_env"
+    env_0.mkdir(parents=True)
+    conda_meta_env_0 = tmp_path / "0/envs/my_env/conta-meta"
```

Comment:
Typo was from before:

```suggestion
    conda_meta_env_0 = tmp_path / "0/envs/my_env/conda-meta"
```

Reply:
Heh I think the typo is intentional, so the function will pick the 2nd env (conda < conta), but I will fix the typo and update the test.